keymanapp/keyman
# chore(common): Add crowdin strings for Polish

This pulls in the crowdin strings for Polish (language tag `pl`).

**TODO before testing**
- [x] @sgschantz to run the iOS and macOS projects through XCode to add `pl` locale.

## User Testing
Note, the overall crowdin progress on Polish is 98%, so there will still be a few untranslated strings that show in English.

* **TEST_ANDROID**
1. Load the PR build of Keyman for Android
2. From the Keyman "Settings" menu, change the "Display Language" to "Polski (Polish)"
3. Return to the Keyman app and verify the strings are in Polish

* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Polish
3. Return to Keyman and verify strings are in Polish

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the device locale to Polish
3. Return to Keyman and verify strings are in Polish.
